contracts/utils/UniversalResolver.sol

```diff
@@ -510,6 +522,11 @@ contract UniversalResolver is ERC165, Ownable {
         bool isCallback = multicallData.name.length == 0;
         bool hasExtendedResolver = _hasExtendedResolver(multicallData.resolver);
 
+        require(
+            !multicallData.isWildcard || hasExtendedResolver,
+            "UniversalResolver: Wildcard on legacy resolvers is not supported"
```

Comment:
Maybe something like "UniversalResolver: Wildcard on non-extended resolvers is not supported"